### DISCUSSION TITLE
Home brew installation pull request #12803

### DISCUSSION THREAD

**Discussion by DerekSpecial** [Homebrew/discussions#2878] [Getting started]
HEAD is now at 2ae21a582 Merge pull request #12803 from Homebrew/dependabot/bundler/Library/Homebrew/zeitwerk-2.5.4
 this is the issue is coming please help to resolve it

**Comment by confucianism72** [comment]
I met this issue too.Have you addressed it ?

**Reply by confucianism72** [reply]
Terminal said the following things then:

fatal: unable to access 'https://github.com/Homebrew/brew/': LibreSSL SSL_connect: Operation timed out in connection to github.com:443

**Reply by gromgit** [reply]
That's a network issue. See https://github.com/Homebrew/discussions/discussions/666 for various causes and suggested fixes.
